app/views/coronavirus_form/offer_other_support.html.erb

```diff
@@ -0,0 +1,29 @@
+<% content_for :title do %><%= t('coronavirus_form.offer_other_support.title') %><% end %>
+<% content_for :meta_tags do %>
+  <meta name="description" content="<%= t('coronavirus_form.offer_other_support.title') %>" />
+<% end %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+<% end %>
+
+<%= render "coronavirus_form/validation_error" %>
+<%= form_tag({},
+  "data-module": "track-coronavirus-form-offer-other-support",
+  "data-question-key": "offer_other_support",
+  "id": "offer_other_support"
+) do %>
+
+<%# Terrible hack for now, will move this into the component properly before launch %>
```

Comment:
Updating govuk_publishing_component to 21.32.0 will provide support for this use case. Example here: https://components.publishing.service.gov.uk/component-guide/textarea/with_label_as_page_heading.

Reply:
thanks very much for that alex, have amended it!